src/lib/core/ripple/ripple.md

```diff
@@ -0,0 +1,102 @@
+# matRipple
+
+Connect user input to screen reactions by using ripples to both indicate the point of touch, and to
+confirm that touch input was received. For touch or mouse, this occurs at the point of contact.
+
+The `matRipple` attribute directive defines an area in which a ripple animates on user interaction.
+
+```html
+<div matRipple [matRippleColor]="myColor">
+  <ng-content></ng-content>
+</div>
+```
+
+By default, a ripple is activated when the host element of the `matRipple` directive receives
+mouse or touch events. Once element is being pressed, a ripple starts fading in from the point
```

Comment:
"Once element is being pressed..." ->
"Upon being pressed, a ripple will begin fading in from the point of contact, radiating to cover the host element"